guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,42 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+You can customize events, payloads, HTTP authentication, content type and headers through {ProjectWebUI}. 
```

Comment:
```suggestion
You can customize events, payloads, HTTP authentication, content type, and headers through {ProjectWebUI}. 
```

See IBM Style Guide: 'Commas between items in a series'. Can you confirm this @melcorr ?